src/components/AdminRoute.js

```
@@ -1,13 +1,21 @@
 import React from 'react';
 import { Route, Redirect } from 'react-router-dom';
-import { auth } from '../firebase';
+import { auth, db } from '../firebase';
 import { useAuthState } from 'react-firebase-hooks/auth';
+import { useDocumentOnce } from 'react-firebase-hooks/firestore';
+import { LinearProgress } from '@material-ui/core';
 
 export const AdminRoute = ({ component: Component, ...rest }) => {
-  const [user, initialising, error] = useAuthState(auth);
+  const [user, initialising] = useAuthState(auth);
```

I'm okay with ignoring this error because it'll probably be surfaced in the error below